interface.go

```diff
@@ -23,3 +23,8 @@ type Repository interface {
 	// Gets the Events which should follow on from the specified channel and event id.
 	Replay(channel, id string) chan Event
 }
+
+type Logger interface {
```

Comment:
I'm inclined to make `Logger` an unexported interface and just define it separately in go-client.  Does that still sound reasonable to you, @ZiaoGeorgeJiang?  Thanks again for submitting this.  Sorry to take so long to look at it.

Reply:
Changed my mind about not exporting it.  I think that isn't in the spirit of having a public interface.  This looks good as is.  THanks.